app/pages/resource/resource-info/ResourceInfo.js

```diff
@@ -94,8 +94,8 @@ function ResourceInfo({
       {!isEmpty(accessibilitySummariesAndDescription) && (
         <ResourcePanel header={t('ResourceAccessibility.headingText')}>
           <ul>
-            {accessibilitySummariesAndDescription.map(accessibility => (
-              <li>{accessibility}</li>
+            {accessibilitySummariesAndDescription.map((accessibility, index) => (
+              <li key={index}>{accessibility}</li>
```

Comment:
Off-topic: this component was raising a console error due to lack of `key` attribute in loop.